app/javascript/articles/Article.jsx

```diff
@@ -2,141 +2,119 @@ import { h } from 'preact';
 import PropTypes from 'prop-types';
 import { articlePropTypes } from '../src/components/common-prop-types/article-prop-types';
 import {
+  ArticleCoverImage,
   CommentsCount,
+  CommentsList,
   ContentTitle,
-  OrganizationHeadline,
-  PublishDate,
-  ReadingTime,
+  Meta,
   SaveButton,
   SearchSnippet,
   TagList,
   ReactionsCount,
+  ReadingTime,
+  Video,
 } from './components';
 import { PodcastArticle } from './PodcastArticle';
 
-/* global timeAgo */
-
 export const Article = ({
   article,
-  currentTag,
+  isFeatured,
   isBookmarked,
-  reactionsIcon,
-  commentsIcon,
-  videoIcon,
   bookmarkClick,
 }) => {
   if (article && article.type_of === 'podcast_episodes') {
     return <PodcastArticle article={article} />;
   }
 
-  const timeAgoIndicator = timeAgo({
-    oldTimeInSeconds: article.published_at_int,
-    formatter: (x) => x,
-  });
-
   return (
-    <div
-      className="single-article single-article-small-pic"
+    <article
+      className={`crayons-story cursor-pointer ${
+        isFeatured && 'crayons-story--featured'
+      }`}
+      id={isFeatured && 'featured-story-marker'}
+      data-featured-article={`articles-${article.id}`}
```

Comment:
```suggestion
      data-featured-article={isFeatured && 'articles-${article.id}`}
```

Reply:
Can you explain this suggestion @nickytonline ?

Reply:
The suggestion is so that the `data-featured-article` only render in markup if it's a featured article, much like the check in the `id`, `id={isFeatured && 'featured-story-marker'}`